tests/offload_utils.hpp

```diff
@@ -23,7 +30,23 @@ if (!(LHS == RHS)) { \
   errors[0]++; \
 }
 
-#ifdef _MDSPAN_HAS_CUDA
+#if defined(_MDSPAN_HAS_CUDA) || defined(_MDSPAN_HAS_HIP)
+
+#if defined(_MDSPAN_HAS_CUDA)
+void deviceSynchronize() { (void) cudaDeviceSynchronize(); }
+template<class T>
+void mallocManaged(T** ptr, size_t size) { (void) cudaMallocManaged(ptr, size); }
+template<class T>
+void freeManaged(T* ptr) { (void) cudaFree(ptr); }
```

Comment:
partly, partly making clear that this is managed memory equivalent and that it needs to be that, because we use the allocations from both host and device in the tests.